shared/src/metabase/shared/util/currency.cljc

```diff
@@ -0,0 +1,851 @@
+          :decimal_digits 2,
+          :rounding 0,
+          :code "BDT",
+          :name_plural "Bangladeshi takas"}],
+   [:BGN {:symbol "BGN",
+          :name "Bulgarian Lev",
+          :symbol_native "лв.",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "BGN",
+          :name_plural "Bulgarian leva"}],
+   [:BHD {:symbol "BD",
+          :name "Bahraini Dinar",
+          :symbol_native "د.ب.‏",
+          :decimal_digits 3,
+          :rounding 0,
+          :code "BHD",
+          :name_plural "Bahraini dinars"}],
+   [:BIF {:symbol "FBu",
+          :name "Burundian Franc",
+          :symbol_native "FBu",
+          :decimal_digits 0,
+          :rounding 0,
+          :code "BIF",
+          :name_plural "Burundian francs"}],
+   [:BND {:symbol "BN$",
+          :name "Brunei Dollar",
+          :symbol_native "$",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "BND",
+          :name_plural "Brunei dollars"}],
+   [:BOB {:symbol "Bs",
+          :name "Bolivian Boliviano",
+          :symbol_native "Bs",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "BOB",
+          :name_plural "Bolivian bolivianos"}],
+   [:BRL {:symbol "R$",
+          :name "Brazilian Real",
+          :symbol_native "R$",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "BRL",
+          :name_plural "Brazilian reals"}],
+   [:BTC {:symbol "BTC",
+          :name "Bitcoin",
+          :symbol_native "BTC",
+          :decimal_digits 8,
+          :rounding 0,
+          :code "BTC",
+          :name_plural "Bitcoins"}],
+   [:BWP {:symbol "BWP",
+          :name "Botswanan Pula",
+          :symbol_native "P",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "BWP",
+          :name_plural "Botswanan pulas"}],
+   [:BYR {:symbol "BYR",
+          :name "Belarusian Ruble",
+          :symbol_native "BYR",
+          :decimal_digits 0,
+          :rounding 0,
+          :code "BYR",
+          :name_plural "Belarusian rubles"}],
+   [:BZD {:symbol "BZ$",
+          :name "Belize Dollar",
+          :symbol_native "$",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "BZD",
+          :name_plural "Belize dollars"}],
+   [:CDF {:symbol "CDF",
+          :name "Congolese Franc",
+          :symbol_native "FrCD",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "CDF",
+          :name_plural "Congolese francs"}],
+   [:CHF {:symbol "CHF",
+          :name "Swiss Franc",
+          :symbol_native "CHF",
+          :decimal_digits 2,
+          :rounding 0.05,
+          :code "CHF",
+          :name_plural "Swiss francs"}],
+   [:CLP {:symbol "CL$",
+          :name "Chilean Peso",
+          :symbol_native "$",
+          :decimal_digits 0,
+          :rounding 0,
+          :code "CLP",
+          :name_plural "Chilean pesos"}],
+   [:CNY {:symbol "CN¥",
+          :name "Chinese Yuan",
```

Comment:
official name is renminbi

Reply:
(this is kind of an oversimplication: renminbi is name of currency, yuan is measure of renminbi in actuality. but we have pound sterling as name of the pound, so we should have renminbi as the name)